applications/launchpad/gui-react/src/components/TBot/TBotPrompt/styles.ts

```diff
@@ -6,28 +6,29 @@ export const PromptContainer = styled(animated.div)<{ $floating?: boolean }>`
   right: 40px;
   bottom: 40px;
   z-index: 1;
-  width: ${({ $floating }) => ($floating ? '476px' : '100%')};
+  width: ${({ $floating }) => ($floating ? '476px' : '692px')};
   height: fit-content;
   display: flex;
   flex-direction: column;
   justify-content: center;
-  align-items: center;
 `
 
-export const ContentRow = styled(animated.div)`
-  width: 90%;
+export const ContentRow = styled(animated.div)<{ $floating?: boolean }>`
+  width: ${({ $floating }) => ($floating ? '417px' : '628px')};
   display: flex;
   flex-direction: row;
   justify-content: flex-start;
 `
 
 export const ContentContainer = styled(animated.div)<{ $floating?: boolean }>`
-  width: ${({ $floating }) => ($floating ? '417px' : '100%')};
+  display: flex;
+  justify-content: center;
+  /* width: ${({ $floating }) => ($floating ? '417px' : '628px')}; */
   height: fit-content;
   margin-right: 30px;
   border-radius: ${({ theme }) => theme.borderRadius(2)};
   /* hard-code required here */
-  background-color: #20053d05;
+  background-color: ${({ $floating }) => ($floating ? '#20053d05' : '')};
```

Comment:
I think that `background-color: ;` is not a valid CSS rule, and at some point we may find some warnings in console log or dev tools. So you can do:

a) use `transparent`:

```
background-color: ${({ $floating }) => ($floating ? '#20053d05' : 'transparent')};
```

b) ... or make background color optional:

```
${({ $floating }) => ($floating ? 'background-color: #20053d05;' : '')};
```

_see line 42 with `top` style_

Reply:
option (b) added.